### PR TITLE
CHORE / General Enhancements

### PR DESCRIPTION
**Pull request by braxeatssnacks**
A couple of general enhancements to the `.dotfiles` _\~ e x p e r i e n c e \~_.

* remove automatic submodule pulls and updates if you're developing `.dotfiles` on a feature branch
* `set inccommand=nosplit` by default for real time updates automatically

##### BUGFIXES
* Restore pane order swapping config in `tmux` 
    * Ctrl ^  Shift ⇧  ArrowRight ⟶ (Move Right)
    * Ctrl ^ Shift ⇧ ArrowLeft ⟵ (Move Left)
* Fix Javascript / JSX syntax highlighting + lint
* Require `scm breeze` internal scripts

### CODE REVIEW

File: vim/init.vim [hunk 1]
```diff
@@ -198,19 +200,19 @@ augroup END
...
+"   \'jsx': ['javascript', 'javascriptreact']
 " \}
 let g:ale_linters = {
-  \'less': ['stylelint'],
   \'javascript': ['eslint'],
-  \'jsx': ['eslint', 'stylelint'],
-  \'python': ['flake8'],
+  \'javascriptreact': ['eslint','stylelint'],
+  \'less': ['stylelint'],
+  \'python': ['black'],
```
Comment (NW-MLakin) [inline vim/init.vim]:
```suggestion
  \'python': ['flake8'],
```
black is only an ale fixer, not a linter
```
❯ ls ~/.config/nvim/plugged/ale/ale_linters/python | sort                                                                                                                                                                  
bandit.vim
flake8.vim
index.html
mypy.vim
prospector.vim
pycodestyle.vim
pydocstyle.vim
pyflakes.vim
pylama.vim
pylint.vim
pyls.vim
pyre.vim
vulture.vim
```

File: vim/init.vim [hunk 2]
```diff
@@ -251,6 +253,8 @@ set wildignore+=*.bmp,*.png,*.jpg,*.jpeg,*.gif
 set wildignore+=*.so,*.swp,*.zip,*.bz2
 " allow unsaved buffers to go into the background
 set hidden
+" real-time replacement
+set inccommand=nosplit
```
Comment (NW-MLakin) [inline vim/init.vim:257]: :airhorn:
:airhorn:
:airhorn:

**Review by NW-MLakin — CHANGES_REQUESTED** [review]
Looks good besides the ale linter for python!

**Review by mmlakin — APPROVED** [review]
Ignore that fucker NW-MLakin, looks good to me!

**Pull request merged**